src/components/forms/formr-part-b/viewSections/ViewSection1.cy-test.tsx

```diff
@@ -0,0 +1,89 @@
+import { mount } from "@cypress/react";
+import { Provider } from "react-redux";
+import { Router } from "react-router-dom";
+import { submittedFormRPartBs } from "../../../../mock-data/submitted-formr-partb";
+import store from "../../../../redux/store/store";
+import ViewSection1 from "./ViewSection1";
+import history from "../../../navigation/history";
+import ViewSectionShouldIncludeThisData, {
+  ISectionSomeDataField
+} from "./ViewSectionTestHelper";
+
+const makeSectionEditButton = (section: number) => {
+  return false;
+};
+
+const formDataToDisplay: ISectionSomeDataField[] = [
```

Comment:
This now uses a single interface for all useable fields. It reads simpler, but it does remove a 'guardrail' against including a field that is not actually part of the applicable set of fields, e.g. including a field from the Work sub-component in a test that examines the main FormRPartB component.

Reply:
now reverted somewhat